quill-core/src/main/scala/io/getquill/quotation/Parsing.scala

```diff
@@ -272,6 +273,15 @@ trait Parsing {
       case "!=" => EqualityOperator.`!=`
     }
 
+  val optionSingleOrderOperationParser: Parser[Operation] = Parser[Operation] {
+    case q"$a.nonEmpty" if is[Option[Any]](a) =>
+      BinaryOperation(astParser(a), EqualityOperator.`!=`, NullValue)
+    case q"$a.isDefined" if is[Option[Any]](a) =>
+      BinaryOperation(astParser(a), EqualityOperator.`!=`, NullValue)
+    case q"$a.isEmpty" if is[Option[Any]](a) =>
+      BinaryOperation(astParser(a), EqualityOperator.`==`, NullValue)
```

Comment:
@fwbrasil But `NullValue` is actually in `quill-core`. If a `submodule` doesn't support `null`, it actually could not handle `option` type. So I think `null comparison` and `Option exists check` are the same thing. Thus it is reasonable to alias those operation to `null comparison`.  Do you have an example indicates that `null comparison` differs from `option exists check`?